### PR TITLE
Just add some lines to file horse

### PR DESCRIPTION
**Pull request by XiangRn**
check some new code lines...

### CODE REVIEW

File: Horse.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 function Horse(){	
 }
+"Today is a great day"
```
Comment (XiangBuss) [inline Horse.js]: This line we should remove them

File: challengebook.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function challengebook(){
+	this.stomach=[];
+}
```
Comment (XiangBuss) [inline challengebook.js:3]: Should we write down some notes here?

File: Horse.js (hunk 1 above)
Comment (XiangRn) [reply Horse.js]: ok! I will fix them, thanks sister

Comment (XiangBuss) [reply Horse.js]: No problem. Everything's ok

File: challengebook.js (hunk 2 above)
Comment (XiangRn) [reply challengebook.js:3]: ok, I will write somethings to note them

**Pull request merged**